src/Sylius/Bundle/AdminBundle/Resources/views/PromotionCoupon/BreadCrumb/generateBreadCrumb.html.twig

```diff
@@ -0,0 +1,11 @@
+{% import 'SyliusAdminBundle:Crud/Macro:breadCrumb.html.twig' as breadCrumb %}
```

Comment:
Breadcrumb is a single world, so I'd change `breadCrumb` to `breadcrumb` here & everywhere else.